tests/expected/report/verification-time/main.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Checks that the time spent in CBMC is reported as "Verification time: <>s"
```

Comment:
The actual check is just that there is a line that says `Verification Time:`, correct?

Reply:
Yes. I'll clarify the comment a bit.